src/Cell.cpp

```diff
@@ -388,6 +506,88 @@ void Cell::setFill(Universe* fill) {
 
 
 /**
+ * @brief Set the Cell's rotation angles about the x, y and z axes.
+ * @details This method is a helper function to allow OpenMOC users to assign
+ *          the Cell's rotation angles in Python. A user must initialize a
+ *          length 3 NumPy array as input to this function. This function then
+ *          stores the data values in the NumPy array in the Cell's rotation
+ *          array. An example of how this function might be called in Python
+ *          is as follows:
+ *
+ * @code
+ *          rotation = numpy.array([0., 0., 90.])
+ *          cell = openmoc.Cell()
+ *          cell.setRotation(rotation)
+ * @endcode
+ *
+ * @param rotation the array of rotation angles
+ * @param num_axes the number of axes (this must always be 3)
+ */
+void Cell::setRotation(double* rotation, int num_axes) {
```

Comment:
Googling 3D rotations, it looks like there are a few different notations in common use [[LINK]](http://mathworld.wolfram.com/EulerAngles.html). Could you add some commentary in the details on which notation is being used and perhaps a reference for where to find more info on this notation?